server/service/client.go

```diff
@@ -100,7 +100,7 @@ func (c *Client) doContextWithBodyAndHeaders(ctx context.Context, verb, path, ra
 		ctx,
 		verb,
 		c.url(path, rawQuery).String(),
-		bytes.NewBuffer(bodyBytes),
+		io.NopCloser(bytes.NewBuffer(bodyBytes)), // TODO: discuss
```

Comment:
@roperzh, let's discuss this when you have a minute.

Reply:
I'm free! want to discuss here? send a link in Slack?